content/webapp/views/components/EventCard/index.tsx

```diff
@@ -41,6 +41,9 @@ export function getLocationText(
   isOnline?: boolean,
   places?: PlaceBasic[] | EventDocumentPlace[]
 ): string | undefined {
+  if (!isOnline && (!places || places.length === 0)) {
+    return undefined;
+  }
```

Comment:
Currently it displays "Online", even if `isOnline` is `false`. This removes all text if nothing is set.